### PR TITLE
PIPRES-261: shipping option module setting

### PR DESCRIPTION
**Pull request by mandan2**
Configuration for recurring order carrier option.

image
image

### CODE REVIEW

File: src/Builder/FormBuilder.php [hunk 1]
```diff
@@ -821,4 +828,54 @@ private function getSettingTabs($isApiKeyProvided)
...
+
+        $mappedCarriers = [];
+
+        $mappedCarriers[] = [
+            'id' => 0,
+            'name' => $this->module->l('Not selected', self::FILE_NAME),
+        ];
+
+        foreach ($carriers as $carrier) {
+            $mappedCarrier = [];
```
Comment (tomjas1997) [inline src/Builder/FormBuilder.php:848]: $mappedCarriers[] = ['id' => $carrier->id, 'name' => $carrier->name];

File: views/templates/admin/Subscription/subscriptions-faq.html.twig [hunk 2]
```diff
@@ -58,6 +58,24 @@
...
 
+  <div class="row">
+    <div class="col-lg-12">
+      <div class="card">
+        <h3 class="card-header">
+          <i class="material-icons">info_outline</i> {{ carrierInformationTitle }}
+        </h3>
+        <div class="card-block">
+          <div class="card-text">
+            <p>{{carrierInformation1}}</p>
```
Comment (tomjas1997) [inline views/templates/admin/Subscription/subscriptions-faq.html.twig:69]: escape for validation

File: views/templates/admin/Subscription/subscriptions-faq.html.twig [hunk 3]
```diff
@@ -58,6 +58,24 @@
     </div>
   </div>
 
+  <div class="row">
+    <div class="col-lg-12">
+      <div class="card">
+        <h3 class="card-header">
+          <i class="material-icons">info_outline</i> {{ carrierInformationTitle }}
```
Comment (tomjas1997) [inline views/templates/admin/Subscription/subscriptions-faq.html.twig:65]: seems incorrect

Comment (mandan2) [reply views/templates/admin/Subscription/subscriptions-faq.html.twig:65]: It's copied from Presta so all good

**Pull request merged**